tracecat/actions/integrations/iam/okta.py

```diff
@@ -28,6 +28,49 @@
 """
 
 
+def create_okta_client() -> httpx.AsyncClient:
+    OKTA_API_TOKEN = os.getenv("OKTA_API_TOKEN")
+    if OKTA_API_TOKEN is None:
+        raise ValueError("OKTA_API_TOKEN is not set")
+    client = httpx.AsyncClient(
+        base_url=os.getenv("OKTA_BASE_URL"),
```

Comment:
you can make the base_url include `/api/v1` so you don't have to include that in downstream usage. So later on you can just pass `/users` as the url in the client